scintillometry/io/psrfits/core.py

```diff
@@ -1,18 +1,20 @@
 # Licensed under the GPLv3 - see LICENSE
 """Interfaces for dealing with PSRFITS fold-mode data."""
+import os
 
 from astropy.io import fits
 from astropy import log
 from collections import defaultdict
 
 from scintillometry.base import BaseTaskBase
-from .hdu import HDU_map
+from .hdu import HDU_map, subint_map
 
 
-__all__ = ['open', 'get_readers', 'PSRFITSReader']
+__all__ = ['open', 'get_readers', 'get_writer', 'PSRFITSReader',
+           'PSRFITSWriter']
 
 
-def open(filename, mode='r', **kwargs):
+def open(filename, mode='r', primary_hdu=None, overwrite=False, **kwargs):
```

Comment:
If we do `overwrite` at all, let's put it in `kwargs`, since it is not relevant for the reader.

Reply:
removed